misc-utils/lsfd-unkn.c

```diff
@@ -497,6 +497,8 @@ static int intcmp(const void *a, const void *b)
 static void anon_eventpoll_attach_xinfo(struct unkn *unkn)
 {
 	struct anon_eventpoll_data *data = (struct anon_eventpoll_data *)unkn->anon_data;
+	if (!data->tfds)
+		return;
 	qsort(data->tfds, data->count, sizeof(data->tfds[0]),
 	      intcmp);
 	if (data->count > 0)
```

Comment:
We can expect data->tfds not to be NULL when data->count is greater than 0.
So we can merge the blocks guarded by the conditions, `(!data->tfds)` and `(data->count > 0)` like:

```
diff --git a/misc-utils/lsfd-unkn.c b/misc-utils/lsfd-unkn.c
index 83367d3ce..2179133e9 100644
--- a/misc-utils/lsfd-unkn.c
+++ b/misc-utils/lsfd-unkn.c
@@ -497,11 +497,13 @@ static int intcmp(const void *a, const void *b)
 static void anon_eventpoll_attach_xinfo(struct unkn *unkn)
 {
 	struct anon_eventpoll_data *data = (struct anon_eventpoll_data *)unkn->anon_data;
-	qsort(data->tfds, data->count, sizeof(data->tfds[0]),
-	      intcmp);
-	if (data->count > 0)
+
+	if (data->count > 0) {
+		qsort(data->tfds, data->count, sizeof(data->tfds[0]),
+		      intcmp);
 		list_add_tail(&data->siblings,
 			      &unkn->file.proc->eventpolls);
+	}
 }
 
 static char *anon_eventpoll_make_tfds_string(struct anon_eventpoll_data *data,
```

Reply:
Good point, done!